testprojects/src/scala/org/pantsbuild/testproject/mutual/BUILD

```diff
@@ -0,0 +1,7 @@
+scala_library()
+
+jvm_binary(
+    name = "bin",
+    main = "org.pantsbuild.testproject.mutual.A",
+    dependencies = [":mutual"],
+)
```

Comment:
This doesn't seem to test the situation when one Rsc-based compilation depends on another. Perhaps, that was intended, but I figured it'd make sense to note this.

Reply:
no, this just covers the mutually dependent files case. I should add one for multiple rsc compiles too